### ISSUE TITLE
Build SPUDlib on Mac OS X Yosemite (10.10.2)

### ISSUE THREAD

**Issue by gmadkat** [iptube/SPUDlib#22]

**Comment by palerikm** [comment]
As a xcode project?

**Comment by gmadkat** [comment]
No, I am not able to bootstrap when I use brew to install dependencies.

**Comment by gmadkat** [comment]
Error when brew is used instead of ports:
bash-3.2$ ./bootstrap.sh
Can't exec "libtoolize": No such file or directory at
/usr/local/share/autoconf/Autom4te/FileUtils.pm line 345, <GEN2> line 5.
autoreconf: failed to run libtoolize: No such file or directory
autoreconf: libtoolize is needed because this package uses Libtool

Workaround: symlink got me past the issue
sudo ln -s /usr/local/bin/glibtoolize /usr/local/bin/libtoolize

**Comment by palerikm** [comment]
That is frustrating. Add a line or two in the README.md file?

**Comment by gmadkat** [comment]
Would you like me to do that in my fork and branch and submit a pull
request?

**Comment by palerikm** [comment]
That would be good.
